esrally/mechanic/launcher.py

```diff
@@ -232,9 +164,9 @@ def start(self, node_configurations):
             nodes.append(cluster.Node(p, host_name, node_name, t))
         return nodes
 
-    def _start_process(self, cmd, node_name):
-        return _start(subprocess.Popen(shlex.split(cmd),
-                                       stdout=subprocess.PIPE, stderr=subprocess.STDOUT, stdin=subprocess.DEVNULL), node_name)
+    def _start_process(self, cmd):
+        subprocess.Popen(shlex.split(cmd), close_fds=True)
```

Comment:
This starts a child process and given that we start `docker-compose` without `-d` in line `156` this means that when the specific Rally command is over then docker-compose will be stopped too.